src/devtools/client/debugger/src/utils/tabs.d.ts

```diff
@@ -0,0 +1,6 @@
+export interface Tab {
+  url: string;
+  sourceId: string | null;
+}
```

Comment:
Can we name this somehow to carry a bit more context, like `EditorTab` or `OpenedEditorTab` etc, because just "Tab" is very abstract?

Reply:
👍